### PR TITLE
WIP: Add pytest-benchmark recipe (2nd attempt)

### PR DESCRIPTION
**Pull request by Juanlu001**
To support #214, supersedes #223.

### CODE REVIEW

**Comment by Juanlu001** [comment]
Rebased to account for the latest changes.

File: recipes/pytest-benchmark/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (Juanlu001) [inline recipes/pytest-benchmark/bld.bat]: No, it wasn't intentional :sweat_smile: I will change it.

File: recipes/pytest-benchmark/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - setuptools
+    - pytest >=2.6
+
+test:
+  imports:
+    - pytest_benchmark
+
+about:
+  home: https://github.com/ionelmc/pytest-benchmark
+  license: BSD
```
Comment (Juanlu001) [inline recipes/pytest-benchmark/meta.yaml]: 2 clause, I will change it.

**Comment by ocefpaf** [comment]
Thanks @Juanlu001!

**Pull request merged**